src/python/bin/aqueduct

```diff
@@ -519,6 +513,13 @@ if __name__ == "__main__":
         action="store_true",
         help="Use this option to expose the server to the public.",
     )
+    start_args.add_argument(
+        "--verbose",
+        default=False,
+        action="store_true",
+        help="""Use this option to see all logs in the terminal. By default, only error logs are 
```

Comment:
English nit: This sounds a little funny for a help message. I'd say something like , "If set to true, all log messages will be printed on stdout. By default, only error and warning-level logs are routed to stdout. All log messages can be found at $HOME/.aqueduct/logs/server".